qa/automation/Alleva 2405/Sign_in_to_app_with_Google_account.feature

```diff
@@ -0,0 +1,15 @@
+Feature: 1.1. Sign in to App with Google account
+
+Scenario: Sign in with Google account granted
+	Given user is on https://qa.ddso-spot.quantori.com/ page
```

Comment:
each 'given' step puts the system in a well-defined step before it will be tested (but i suppose it could be moved to the Background since it's used in both scenarios)

Reply:
I'd love to move it to background for both tests, however, it was undesirable to start Scenario: with ' when' which would have been the case then

Reply:
@AlexandraPopovaz  1) condition about to start each scenario with 'Given' was specified in the chat 2) user is not logged in, he only has page open in front of him, ' log-in' is to be tested in scenario